internal/manifests/querier/querier.go

```diff
@@ -79,6 +87,24 @@ func deployment(params manifestutils.Params) (*v1.Deployment, error) {
 							Name:  "tempo",
 							Image: tempo.Spec.Images.Tempo,
 							Args:  []string{"-target=querier", "-config.file=/conf/tempo.yaml"},
+							Env: []corev1.EnvVar{
```

Comment:
These values are the same than in `PatchTracingJaegerEnv` right? Would it make sense to place them in a common location?